### PR TITLE
fix: logged errors are reported to error reporting

### PR DESCRIPTION
**Pull request by DominicKramer**
Fixes: #32

### CODE REVIEW

File: system-test/logging-winston.ts [hunk 1]
```diff
@@ -114,4 +115,38 @@ describe('LoggingWinston', () => {
...
+    afterEach(async () => {
+      await errorsTransport.deleteAllEvents();
+    });
+
+    it('reports errors when logging errors', function(done) {
+      this.timeout(2*ERROR_REPORTING_DELEY_MS);
+      const message = `an error at ${Date.now()}`;
+      // logger does not have index signature.
+      // tslint:disable-next-line:no-any
+      (logger as any)['error'].apply(logger, ['an error', new Error(message)]);
```
[the remark this answers is not in the document]
Comment (DominicKramer) [reply system-test/logging-winston.ts]: This is how the rest of the system tests are written.

File: src/index.ts [hunk 2]
```diff
@@ -106,9 +106,17 @@ export class LoggingWinston extends winston.Transport {
...
+    this.serviceContext = options.serviceContext || {};
     this.prefix = options.prefix;
     this.labels = options.labels;
+
+    // serviceContext.service is required by the Error Reporting
+    // API.  Without it, errors that are logged with level 'error'
+    // or higher will not be displayed in the Error Reporting
+    // console
+    if (!this.serviceContext.service) {
+      this.serviceContext.service = 'default';
```
Comment (ofrobots) [inline src/index.ts]: Why is 'default' a good value? If the user provided a `serviceContext` but not value for `service`, shouldn't we throw?

Comment (ofrobots) [reply src/index.ts]: If the user *intended* to use error reporting, but configured it incorrectly, we should throw.

When trying to correlate errors – that happen rarely – people may not necessarily know that `default` and `unknown` is the service they should look up.

Is there precedent from other languages you can use here?

**Comment by ofrobots** [comment]
In order to avoid duplicate discussion, lets continue discussion on https://github.com/googleapis/nodejs-logging-bunyan/pull/122 and wait until that is resolved before proceeding here.

**Comment by DominicKramer** [comment]
@ofrobots and I discussed this PR in person and, after updating this PR to not modify the default section of the README, this PR is ready to land.

**Pull request merged**